packages/common/src/content/_internal/ContentUiSchemaEdit.ts

```diff
@@ -52,6 +52,7 @@ export const buildUiSchema = async (
             options: {
               control: "hub-field-input-input",
               type: "textarea",
+              rows: 4,
```

Comment:
Should we change this to be the default in hub-components but still respect an override? Might be good so we don't have to always duplicate this option.

Reply:
Maybe... the only reason I could see that going wrong is if we're using the config editor with textareas elsewhere and it's an unwanted change, but I don't know if we are.

Reply:
@juliannemarik any thoughts? If you agree with Aaron then we can totally leave it as it is

Reply:
Yes, what Aaron said would be my primary concern. @abp6318, I'm also wondering if you were given any instruction to make the same "row" adjustment for the "description" fields? If not, and we want the "description" field to remain as is, we'd essentially need this same PR to override the default of 4.

In any case, I feel like this might be a bit more of a product question as to what they want the default height to be set at for textarea fields.

Reply:
@juliannemarik and @abp6318 good points, let's get this merged then and we can open up a ticket to get more product guidance for the future.